static/app/components/replays/breadcrumbs/openReplayComparisonButton.tsx

```diff
@@ -35,6 +36,7 @@ export function OpenReplayComparisonButton({
       size={size}
       analyticsEventKey="replay.details-hydration-modal-opened"
       analyticsEventName="Replay Details Hydration Modal Opened"
+      analyticsParams={{url: getLocationHref()}}
```

Comment:
This might be hard to group-by later on. something like a transaction name would be better, or when the button is created we could pass in a simple string that names the page. 

like `<OpenReplayComparisonButton surface="replay-breadcrumbs" ... />` or `<OpenReplayComparisonButton surface="issue-details" ... />`

Reply:
Got it, I think there are a lot of components we have to pass this down though.. maybe we can find a cleaner way to get the prefix/suffix of a URL?

Reply:
pathname is everything after the `sentry.io`

Reply:
we usually don't want to log the whole url. it's not important to know that the url was `/issues/3758928546/events/9e0d6bc120794875a08ec471ac8840c0` better to know it was `issues/<issue_id>/events/<event_id>`